CRVResponse/src/CrvStepsFromStepPointMCs_module.cc

```diff
@@ -59,6 +59,8 @@ namespace mu2e
 	  Comment("Starting size for crvIndex-particle vector"),4};
 	fhicl::Atom<bool> removeNeutralParticles{ Name("removeNeutralParticles"),
 	  Comment("Removes steps of neutral particles"),true};
+	fhicl::Atom<bool> useTotalEDep{ Name("useTotalEDep"),
+	  Comment("Use total energy deposition instead. Visible is a new default"),false};
```

Comment:
Normally I prefer that parameters that affect physics not have defaults in code and that they be required fcl parameters.

Is this seen as a temporary feature that will go away after the older CRY files are retired? If we insist on this policy will there be a lot of fcl files to update?

Will this change modify the behaviour of the nightly comics tests?

Reply:
> Normally I prefer that parameters that affect physics not have defaults in code and that they be required fcl parameters.
> 
> Is this seen as a temporary feature that will go away after the older CRY files are retired?
Yes. After the current CRY files are retired we can remove this option.
We plan to use visible energy going forward  

> If we insist on this policy will there be a lot of fcl files to update?
I think it's not a large number, but I'll redirect this question to Ralf.
 
> Will this change modify the behaviour of the nightly comics tests?
No, the default behavior has not changed.
We only use non-standard config (useTotalEDep: true) in JobConfig/cosmic/cosmic_s2_converter.fcl

Reply:
Since this is a transient thing, and you only enable the alternate behaviour in one place,  I am comfortable leaving it as is.  There is no need to modify the other files to explicitly chose the default.

I will resolve this comment.